src/_shared/scss/_palette.scss

```diff
@@ -45,7 +45,7 @@ $comment-support-4:      #e6711b;
 
 // Multimedia palette
 $multimedia-main-1:      #333333;
-$multimedia-main-2:      #ff4e36;
+$multimedia-main-2:      #ffbb00;
```

Comment:
This colour is also used [here](https://github.com/guardian/commercial-templates/blob/24e60cf563b6330560eb5c81a0f2045a07b852f4/src/_shared/scss/_adverts-capi.scss), will this change generated an undesirable side effect?

Reply:
Will check with the people using this styling

Reply:
Hey Guy, sorry this took ages for me to have a look at - changed my git notification settings now! That hex code is also the kicker colour for a Labs supported container:

![image](https://user-images.githubusercontent.com/4101937/40541276-06cab620-6013-11e8-8b02-751b47fd43f7.png)

I can easily just add a different value and hex code to import if it helps - I was in a rush the first time I did this though, it needs a massive clean up to make it a bit clearer. Can we chat when you're back in on Monday?